newIDE/app/public/index.html

```diff
@@ -13,7 +13,8 @@
         margin: 0;
         padding: 0;
         height: 100%;
-        background-color: #f0f0f0;
+        background-color: #000;
```

Comment:
Also check in Electron main.js what color we define for the background, I'm not sure but setting the same might avoid a flash at the opening (can't remember if we wait or not for the page to be at least loaded before showing the window).

Reply:
Good catch!